### PR TITLE
Remove FEATURE_CORE_DLR

### PR DESCRIPTION
**Pull request by slozier**

### CODE REVIEW

File: Src/Microsoft.Scripting/Runtime/LanguageContext.cs [hunk 1]
```diff
@@ -13,16 +13,10 @@
  *
  * ***************************************************************************/
 
-#if FEATURE_CORE_DLR
 using System.Linq.Expressions;
 #if !WIN8
```
Comment (slide) [inline Src/Microsoft.Scripting/Runtime/LanguageContext.cs]: We can remove WIN8 too I think, but we can make it another PR if desired.

Comment (slozier) [inline Src/Microsoft.Scripting/Runtime/LanguageContext.cs]: I'll leave it for a separate PR.
